api/src/main/java/org/pmiops/workbench/api/EgressEventsAdminController.java

```diff
@@ -28,6 +36,10 @@
   private static final int DEFAULT_PAGE_SIZE = 32;
   private static final int MAX_PAGE_SIZE = 128;
 
+  // Admins can update to/from this set of statuses exclusively.
+  private static final Set<EgressEventStatus> updateableStatuses =
+      ImmutableSet.of(EgressEventStatus.REMEDIATED, EgressEventStatus.VERIFIED_FALSE_POSITIVE);
```

Comment:
Do we want to support update `VERIFIED_FALSE_POSITIVE` to `REMEDIATED`?
Is this should be a `from-to` map? Then the validation would be `(map.has(from) && map.get(from)==to)`

Reply:
Yes, I intentionally am supporting that - this basically lets you undo if you accidentally mark the wrong one.

Reply:
It would still be more extendable once we deicide to add more states here.
e.g., when added the 3rd state, and we only want to support `REMEDIATED -> new state`, having from and to in the mixed set makes the problem harder. 
For now, the map can be `VERIFIED_FALSE_POSITIVE-> REMEDIATED` and `REMEDIATED  -> VERIFIED_FALSE_POSITIVE`

Reply:
I see your point, but I think it's too speculative. It is reasonable to assume that if there are statuses that an admin can set manually, they should be able to freely switch between any of them (e.g. to undo a status change). I think it's likely that this property holds for the lifetime of this code.

If it doesn't hold, then I think the code change you suggest makes sense, and is an easy one to make when/if we need it.